### PR TITLE
Describe HNS Unspecified error

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

File: doc/developer_guide/HNS_error_bestiary.md [hunk 1]
```diff
@@ -78,11 +78,15 @@ The following chapter contains a list of HNS errors encountered throughout devel
...
+1.  When attempting to create a transparent HNS network
+    * when creation of another network or VMSwitch is already in progress
+      (eg. we've just started Docker service and it tries to create the NAT network).
+
+    We can work around this bug by retrying after a few seconds.
 
 ### Reproduction:
 
-TODO
+Try to create multiple transparent HNS networks in a loop simultaneously with multiple processes.
```
Comment (m-kostrzewa) [inline doc/developer_guide/HNS_error_bestiary.md]: Wouldn't it create only a single vSwitch?

Comment (krdln) [reply doc/developer_guide/HNS_error_bestiary.md]: Yep, you're right. I think high load of creating any HNS networks could trigger this bug.

Comment (m-kostrzewa) [reply doc/developer_guide/HNS_error_bestiary.md]: @krdln this should be included in the doc then :)

Comment (m-kostrzewa) [reply doc/developer_guide/HNS_error_bestiary.md]: highload thing pls

**Review by m-kostrzewa — APPROVED** [review]

**Pull request merged**